### PR TITLE
test file added for Order By Token

### LINKED ISSUE #16099: Add unit test for OrderByToken#toString
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### PR DESCRIPTION
**Pull request by shreya024**
Fixes #16099 .

### CODE REVIEW

**Comment by shreya024** [comment]
CI test for JDK 8 - on macos-latest is showing error in a different test file
DataSourcePoolDestroyerTest.assertAsyncDestroyHikariDataSource:40->createHikariDataSource:57 » TestTimedOut

**Comment by strongduanmu** [comment]
> CI test for JDK 8 - on macos-latest is showing error in a different test file DataSourcePoolDestroyerTest.assertAsyncDestroyHikariDataSource:40->createHikariDataSource:57 » TestTimedOut

Got it.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    @Before
+    public void setup() {
+        orderByToken = new OrderByToken(0);
+    }
+
+    @Test
+    public void assertToString() {
+        assertThat(orderByToken.toString(), is(" ORDER BY  "));
+    }
+
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java:39]: @shreya024 Please remove this uesless blank line.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class OrderByTokenTest {
+
+    private OrderByToken orderByToken;
+
+    @Before
+    public void setup() {
+        orderByToken = new OrderByToken(0);
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java:32]: @shreya024 Can you mock OrderByToken with columnLabels and orderDirections?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java (hunk 2 above)
Comment (shreya024) [reply shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java:32]: I'm working on it

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import org.mockito.Mock;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class OrderByTokenTest {
+
+    @Mock
```
Comment (TeslaCN) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java:32]:
```suggestion
```

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        orderDirections = orderByToken.getOrderDirections();
+        columnLabels.add(0, "Test1");
+        columnLabels.add(1, "Test2");
+        orderDirections.add(0, OrderDirection.ASC);
+        orderDirections.add(1, OrderDirection.ASC);
+    }
+
+    @Test
+    public void assertToString() {
+        StringBuilder result = new StringBuilder();
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java:50]: Hi @shreya024, I think we don't need this logic here. We just need assert toString result, like `assertThat(orderByToken.toString(), is("ORDER BY Test1 ASC, Test2 ASC"));`

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java (hunk 4 above)
Comment (shreya024) [reply shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/OrderByTokenTest.java]: Yes,thank you so much for guiding me.

**Review by strongduanmu — APPROVED** [review]
@shreya024 Thank you.

**Pull request merged**